### ISSUE TITLE
Parameter Constant Values/allowableValues/enum do not get generated

### ISSUE THREAD

**Issue by StephenOTT** [micronaut-projects/micronaut-openapi#141]
Consider a Operation on a controller function:

```
    @Operation(
            summary = "...",
            description = "....",
            security = [
                SecurityRequirement(name = "basicAuth")
            ],
            parameters = [
                Parameter(name = Headers.ACCEPT, `in` = ParameterIn.HEADER, required = true, schema = Schema(type = "string", allowableValues = ["some-value"] ))
            ]
    )
```

and the docs show: https://swagger.io/docs/specification/describing-parameters/#constant

But the above code will only generate the following yml

```
      parameters:
      - name: Accept
        in: header
        required: true
        schema:
          type: string
```

**Comment by StephenOTT** [comment]
this appears to be solved in 1.4.0

**Comment by frangarcia** [comment]
I am not sure this have been properly fixed. I am having the same issue with this parameter

`@Parameter(name = 'order', schema = @Schema(implementation = String.class, allowableValues = ["asc","desc"]))`

I have those dependencies

```
io.micronaut.configuration:micronaut-openapi:1.4.1
io.swagger.core.v3:swagger-annotations:2.1.2
```
